frontend/tests/acceptance/edit-person-test.js

```diff
@@ -23,7 +23,7 @@ test('/people/:id edit person data', async function(assert) {
   await page.toggleEditForm();
 
   /* eslint "no-undef": "off" */
-  await selectChoose('#role', 'Controller')
+  await selectChoose('#role', '.ember-power-select-option', 0)
```

Comment:
can't you select by value ?
selectChoose #role, 'Software-Engineer'

Reply:
No that wouldn't work because the roles in the power-select are ember models. You would need the reference to that model in order for it to work like that, which we don't have. So the quickest and easiest way is this.